about/changelog.md

```diff
@@ -8,8 +8,22 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
-## 🤩 SQL Assistant, TimescaleDB v2.17, HIPAA compliance, and better logging
+### 🗝️ IP Allow Lists
```

Comment:
I think you want to promote the header level here so that it shows up in the right sidebar? 

```suggestion
## 🗝️ IP Allow Lists
```

Reply:
Updated